llmware/library.py

```diff
@@ -633,7 +948,21 @@ def install_new_embedding (self, embedding_model_name=None, vector_db=None,
 
     def delete_library(self, library_name=None, confirm_delete=False):
 
-        """ Deletes all artifacts of a library """
+        """ Deletes all artifacts of a library 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Deletes all artifacts of a library.
```